main.tf

```diff
@@ -14,15 +19,18 @@ resource "aws_cloudwatch_event_target" "billing_notifier_lambda_event_target" {
 data "aws_caller_identity" "current" {}
 
 resource "aws_lambda_permission" "billing_notifier_lambda_permission" {
-  statement_id  = "AllowExecutionFromCloudWatch"
-  action        = "lambda:InvokeFunction"
-  function_name = module.billing_notifier_lambda.lambda_function.function_name
-  principal     = "events.amazonaws.com"
-  source_arn    = aws_cloudwatch_event_rule.billing_notifier_lambda_event_rule.arn
+  statement_id   = "AllowExecutionFromCloudWatch"
+  action         = "lambda:InvokeFunction"
+  function_name  = module.billing_notifier_lambda.lambda_function.function_name
+  principal      = "events.amazonaws.com"
+  source_arn     = aws_cloudwatch_event_rule.billing_notifier_lambda_event_rule.arn
+  source_account = data.aws_caller_identity.current.account_id
+
+  tags = var.tags
+
   depends_on = [
     module.billing_notifier_lambda
   ]
-  source_account = data.aws_caller_identity.current.account_id
 }
```

Comment:
This `source_account` was recently added, shouldnt be removed. Perhaps you are based off of an older commit?